source/_components/device_tracker.unifi.markdown

```diff
@@ -68,6 +68,11 @@ ssid_filter:
     type: list of strings
     required: false
     default: None
+monitor_all:
+    description: Pull all available extra attributes from the Unifi controller.
```

Comment:
Maybe extend the description with something like
"Setting this value to `true` overrides the `monitored_conditions` option."